openslides_backend/action/actions/meeting_user/mixin.py

```diff
@@ -10,6 +11,77 @@
 
 
 class MeetingUserMixin(Action):
+    standard_fields = [
+        "comment",
+        "number",
+        "structure_level",
+        "vote_weight",
+        "personal_note_ids",
+        "speaker_ids",
+        "supported_motion_ids",
+        "submitted_motion_ids",
+        "assignment_candidate_ids",
+        "vote_delegated_vote_ids",
+        "vote_delegated_to_id",
+        "vote_delegations_from_ids",
+        "chat_message_ids",
+    ]
+
+    def check_permissions(self, instance: Dict[str, Any]) -> None:
+        """ standard_fields have to be checked, but also it is not permitted, create an empty meeting_value instance"""
```

Comment:
Make this comment more verbose, the meaning is not clear. Generally, some more comments would be good in this method since it combines multiple functionalities and is fairly complex.

Reply:
changed